[CI] MatchOperatorIT testScoring_Zero_OutsideQuery failing

**Build Scans:**
- [elasticsearch-intake #19473 / part3](https://gradle-enterprise.elastic.co/s/z35untoaic7g6)
- [elasticsearch-intake #19469 / part3](https://gradle-enterprise.elastic.co/s/5d3znripdl4gs)
- [elasticsearch-pull-request #60792 / part-3](https://gradle-enterprise.elastic.co/s/spswuw2uzn7ru)
- [elasticsearch-pull-request #60787 / part-3](https://gradle-enterprise.elastic.co/s/rcx5tqg5lxl4k)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:esql:internalClusterTest" --tests "org.elasticsearch.xpack.esql.plugin.MatchOperatorIT.testScoring_Zero_OutsideQuery" -Dtests.seed=B08E5C0B45973F4D -Dtests.locale=tr-TR -Dtests.timezone=America/Fortaleza -Druntime.java=23
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.xpack.esql.plugin.MatchOperatorIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!(testScoring_Zero_OutsideQuery),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: 
Expected: <[[This is a brown fox, 0.0], [The quick brown fox jumps over the lazy dog, 0.0]]>
     but: was <[[The quick brown fox jumps over the lazy dog, 0.0], [This is a brown fox, 0.0]]>
```

**Issue Reasons:**
- [main] 4 failures in test testScoring_Zero_OutsideQuery (57.1% fail rate in 7 executions)
- [main] 2 failures in step part3 (66.7% fail rate in 3 executions)
- [main] 2 failures in step part-3 (50.0% fail rate in 4 executions)
- [main] 2 failures in pipeline elasticsearch-intake (66.7% fail rate in 3 executions)
- [main] 2 failures in pipeline elasticsearch-pull-request (50.0% fail rate in 4 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -333,9 +333,6 @@ tests:
 - class: org.elasticsearch.smoketest.MlWithSecurityIT
   method: test {yaml=ml/3rd_party_deployment/Test start deployment fails while model download in progress}
   issue: https://github.com/elastic/elasticsearch/issues/120814
-- class: org.elasticsearch.xpack.esql.plugin.MatchOperatorIT
-  method: testScoring_Zero_OutsideQuery
-  issue: https://github.com/elastic/elasticsearch/issues/124132
 - class: org.elasticsearch.search.query.QueryPhaseTimeoutTests
   method: testScorerTimeoutPoints
   issue: https://github.com/elastic/elasticsearch/issues/124140

--- a/x-pack/plugin/esql/src/internalClusterTest/java/org/elasticsearch/xpack/esql/plugin/MatchOperatorIT.java
+++ b/x-pack/plugin/esql/src/internalClusterTest/java/org/elasticsearch/xpack/esql/plugin/MatchOperatorIT.java
@@ -221,7 +221,7 @@ public class MatchOperatorIT extends AbstractEsqlIntegTestCase {
     public void testScoring_Zero_OutsideQuery() {
         var query = """
             FROM test METADATA _score
-            | SORT _score DESC
+            | SORT content DESC
             | KEEP content, _score
             """;
 

